tck/src/main/x/tck/tuples/Basic.x

```diff
@@ -7,11 +7,11 @@ class Basic {
         testConstAccess();
         testIndexAccess();
         testEquality();
-        testVoidConv();
-        testIntConv();
+        //testVoidConv();
+        //testIntConv();
         testSlice();
     }
-    
+
     @Test
```

Comment:
Remove whitespace diff to consistent with the rest of the code base, including to Basic.x itself, which now has both len 0 whitespace lines as well as whitespace line with spaces.

Reply:
This is something that we should figure out together how to tackle, as it will save us all a lot of frustration over time (it's not just Marcus who will benefit :)

And good news: Gene says that Cliff has already addressed this problem, which is why this diff exists (and should be a one-time thing).